haystack/preview/components/audio/whisper_remote.py

```diff
@@ -38,11 +38,14 @@ class Input(ComponentInput):
     class Output(ComponentOutput):
         documents: List[Document]
 
-    def __init__(self, api_key: str, model_name: WhisperRemoteModel = "whisper-1"):
+    def __init__(
+        self, api_key: str, model_name: WhisperRemoteModel = "whisper-1", api_base: str = "https://api.openai.com/v1"
+    ):
         """
         Transcribes a list of audio files into a list of Documents.
 
         :param api_key: OpenAI API key.
+        :param api_base: OpenAI base URL, defaults to `"https://api.openai.com/v1"`.
         :param model_name_or_path: Name of the model to use. It now accepts only `whisper-1`.
```

Comment:
Let's fix the order of arguments here and while we're at it use the correct name for the `model_name` parameter.
```suggestion
        :param model_name: Name of the model to use. It now accepts only `whisper-1`.
        :param api_base: OpenAI base URL, defaults to `"https://api.openai.com/v1"`.
```